build.gradle

```diff
@@ -33,6 +33,10 @@ dependencies {
     minecraft "com.mojang:minecraft:${project.minecraft_version}"
     mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
     modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
```

Comment:
While technically out of scope, perhaps this could become modImplementation instead of modCompile